app/views/page-enquiries/partials/enquiries-feed.html

```diff
@@ -21,7 +21,7 @@
                         <strong><span hs-bind="data.customerName">Customer Name</span>'s <span hs-bind="data.word">Booking Word</span></strong>
                         <small>
                             <span hs-bind="data.venueName">No Venue Name</span> &bull;
-                            <span hs-bind="data.budget">No Budget</span> &bull;
+                            <span hs-bind="data.budget | pounds">No Budget</span> &bull;
                             <span hs-bind="data.eventdate | date">No Event Date</span>
```

Comment:
This should be explicit, pls replace empty HTML inside this tag with what's here